packages/babel-types/README.md

```diff
@@ -292,6 +292,14 @@ Aliases: `Flow`, `FlowDeclaration`, `Statement`, `Declaration`
  - `id` (required)
  - `body` (required)
 
+### t.declareModuleExports(typeAnnotation)
+
+See also `t.isDeclareModuleExports(node, opts)` and `t.assertDeclareModuleExports(node, opts)`.
+
+Aliases: `Flow`, `FlowDeclaration`, `Statement`, `Declaration`
+
+ - `typeAnnotation` (required)
+
```

Comment:
No worries; noticed that, while the file contains a generated section, it contains a human-authored header so thought that the appropriate course of action was to push it upstream.
btw. Just familiarising myself with the Babel neighbourhood so just followed you here on GitHub and on Twitter.